modules/ROOT/pages/visualizer-app-network.adoc

```diff
@@ -78,7 +81,7 @@ You can export the view of your network showing on your canvas to a PNG file.
 * If you customized your view with filters, the customized network topology is exported.
 * If you selected metrics or durations, these settings are reflected in the export.
 
-To export a PNG view of your network, click *Export as PNG* in the side panel.
+To export a PNG view of your network, click the *Export as Image* button next to the zoom controls in the bottom right.
```

Comment:
Does the audience know what a PNG view is?

Reply:
I would recommend -- "To export a view of your network as a PNG"

Reply:
I would be shocked if this audience did not know what a PNG is. I accepted Ashley's suggestion.